pkg/webhook/vlanconfig/validator.go

```diff
@@ -90,7 +116,38 @@ func (v *vlanConfigValidator) Resource() types.Resource {
 		ObjectType: &networkv1.VlanConfig{},
 		OperationTypes: []admissionregv1.OperationType{
 			admissionregv1.Create,
+			admissionregv1.Update,
 			admissionregv1.Delete,
 		},
 	}
 }
+
+func (v *vlanConfigValidator) checkOverlaps(vc *networkv1.VlanConfig) error {
+	var matchedNodes []string
+	if vc.Annotations == nil || vc.Annotations[utils.KeyMatchedNodes] == "" {
+		return nil
+	}
+	if err := json.Unmarshal([]byte(vc.Annotations[utils.KeyMatchedNodes]), &matchedNodes); err != nil {
+		return err
+	}
+	if len(matchedNodes) == 0 {
+		return nil
+	}
+
+	pausedNodes := make([]string, 0, len(matchedNodes))
+	for _, node := range matchedNodes {
+		vsName := utils.Name("", vc.Spec.ClusterNetwork, node)
+		if vs, err := v.vsCache.Get(vsName); err != nil && !apierrors.IsNotFound(err) {
+			return err
+		} else if err == nil && vs.Status.VlanConfig != vc.Name {
+			// The vlanconfig is found means a vlanconfig with the same clusternetwork  has been taken effect on this node.
+			pausedNodes = append(pausedNodes, node)
```

Comment:
you should be able to return the error message in here directly.

Reply:
I think it's better to return all the overlap nodes in the error message.

Reply:
I will also consider returning all nodes in one error message here.